Rotator/MaxStorageSizeRotator.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Zenstruck\BackupBundle\Rotator;
+
+use Zenstruck\BackupBundle\Destination\Backup;
+
+final class MaxStorageSizeRotator implements Rotator
+{
+    /**
+     * @var integer
+     */
+    private $maxSize;
+
+    /**
+     * A constructor.
+     *
+     * @param integer $maxSize Maximum storage size.
+     */
+    public function __construct($maxSize)
```

Comment:
Maybe this lib can be an optional requirement: https://github.com/gabrielelana/byte-units

We could check if the value of $maxSize is an integer. If so, just use it. If not, ensure the package above is loaded and parse with it.